src/Project/HCA/client/componentFactory.ts

```diff
@@ -22,7 +20,7 @@ import * as PageContent from 'Feature/PageContent/client';
 import GlobalFooter from 'Feature/Navigation/client/GlobalFooter';
 import GlobalHeader from 'Feature/Navigation/client/GlobalHeader';
 
-const components = new Map<string, React.ComponentType<{}>>();
+const components = new Map<string, any>();
```

Comment:
Are you sure? `any` is not good practice

Reply:
Yes. Our components implement `ComponentClass` interface, so they create `Component` instance from constructor, this type is required from `componentFactory`. But there is no cast between 'ComponentClass` interface and `Component` class. If you know how we can do this, I'll be very appreciate for help.